prisma/migrations/20240226024040_init/migration.sql

```diff
@@ -0,0 +1,93 @@
+-- CreateEnum
+CREATE TYPE "preeshHeaviness" AS ENUM ('LOWKEY', 'HEAVY', 'HEAVYFR', 'SUPERPREESH');
+
+-- CreateTable
+CREATE TABLE "Beast" (
+    "id" SERIAL NOT NULL,
+    "gamerTag" VARCHAR(20) NOT NULL,
+    "email" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Beast_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Preesh" (
+    "id" SERIAL NOT NULL,
+    "text" TEXT NOT NULL,
+    "authorId" INTEGER NOT NULL,
+    "receiverId" INTEGER NOT NULL,
```

Comment:
I believe authorId and receiverId should have foreign keys to Beast table. See https://www.prisma.io/docs/orm/prisma-schema/data-model/relations for documentation on this.